getting-started/capture-workflow.md

```diff
@@ -77,10 +77,10 @@ The default `recordings` directory will have the following hierarchy.
 
 * `recordings`
   * 2016-04-05
-    * 001
-    * 002
-    * 003
-    * ####
+	    * 001
```

Comment:
Why so many spaces? Isn't `.md` syntax just using two whitespaces for indented lists.